### DISCUSSION TITLE
High abundance of H3+ in molecular clouds

### DISCUSSION THREAD

**Discussion by ftessier** [a-cosmology-group/acg#38] [Astrophysics and Physics]
From Nature Chemistry[^1]:

**$\mathbf{\text{D}_3^+}$ formation through photoionization of the molecular $\mathbf{\text{D}_2\text{−}\text{D}_2}$**

> The trihydrogen cation $\text{H}_3^+$ is also a key intermediate in a range of chemical processes in interstellar environments, such as the formation of various organic molecules and early stars. However, the unexpected high abundance of $\text{H}_3^+$ in molecular clouds remains challenging to explain.

> Although $\text{H}_2$ dimers will not be subjected to intense laser pulses in interstellar space, the ionization dynamics can be expected to proceed very similarly for high-energy photons (cosmic rays) due to the electronic simplicity of $\text{H}_2$. Although the detection of $\text{H}_2$ dimers in the interstellar medium is challenging due to their weak bond, the observation of $\text{H}_2$ dimers in the atmospheres of Jupiter and Saturn could indicate that the contribution of hydrogen dimers to the formation of $\text{H}_3^+$ has been neglected.

[^1]: https://www.nature.com/articles/s41557-023-01231-z

**Comment by RedshiftDrift** [comment]
Interesting that such a basic reaction has not been studied in details.  $\text{H}_2$ is a difficult molecule to study.
